service/src/main/java/bio/terra/workspace/db/ResourceDao.java

```diff
@@ -465,6 +470,34 @@ public void createControlledResource(ControlledResource controlledResource)
     storeResource(controlledResource);
   }
 
+  @WriteTransaction
+  public void setPrivateResourceState(
+      ControlledResource resource, PrivateResourceState privateResourceState) {
+    String sql =
+        "UPDATE resource SET private_resource_state = :private_resource_state WHERE workspace_id = :workspace_id AND resource_id = :resource_id AND access_scope = :private_access_scope";
```

Comment:
can you add a dbUtils for building the sql string.

Reply:
I don't think this needs to be built like the `dbUtils` queries do since this is always a final String query.